### PR TITLE
Implement a hello-strongbox-pypi

### LINKED ISSUE #812: Implement a hello-strongbox-pypi example
# Task Description

We need a simple "Hello, World!"-styled application that illustrates:
* [x] The creation of a PyPi package
* [x] Deploying PyPi packages to Strongbox
* [ ] Resolving PyPi packages
* [ ] A detailed step-by-step `README.md` illustrating how to configure things, as well as how to execute the steps above, along with actual successful output

The code should be located under the [strongbox-examples](https://github.com/strongbox/strongbox-examples) project and it should be called `strongbox-pypi`.

It would probably be helpful to have a look at how things have been done in the other examples under the [strongbox-examples](https://github.com/strongbox/strongbox-examples) project.

This is a sub-task of #807. It should be relatively straight-forward to implement this and you should get acquainted with the master task, before proceeding. This task depends on #811, or should be implemented in parallel with it.

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @fuss86

### PR DESCRIPTION
**Pull request by anki2189**
Closes strongbox/strongbox#812 .

This PR should be merged after https://github.com/strongbox/strongbox/pull/1633

### CODE REVIEW

File: hello-strongbox-pypi/LICENSE [hunk 1]
```diff
@@ -0,0 +1,17 @@
+MIT License
```
Comment (carlspring) [inline hello-strongbox-pypi/LICENSE]: This should be an Apache 2.0 license.

Comment (anki2189) [reply hello-strongbox-pypi/LICENSE]: Changed

File: hello-strongbox-pypi/setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="hello_world_pypi",
+    packages = ['hello_world_pypi'],
+    license='MIT',
```
Comment (carlspring) [inline hello-strongbox-pypi/setup.py]: Here as well, please.

Comment (anki2189) [reply hello-strongbox-pypi/setup.py]: Done.

**Comment by anki2189** [comment]
@carlspring Please review and merge this PR.

File: hello-strongbox-pypi/.pypirc.template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+index-servers=
+  strongbox
+  testpypi
+
+[strongbox]
+repository: http://localhost:48080/storages/storage-pypi/pypi-releases
+username: admin
+password: password
+
+[testpypi]
```
Comment (carlspring) [inline hello-strongbox-pypi/.pypirc.template]: I don't think we need this for the sake of the example.

**Comment by carlspring** [comment]
Could you please also:
* [ ] Illustrate how to do with with `twine`
* [ ] Add a page in the wiki for this

Thanks! :)

File: hello-strongbox-pypi/.pypirc.template (hunk 3 above)
Comment (anki2189) [reply hello-strongbox-pypi/.pypirc.template]: Fixed.

**Comment by anki2189** [comment]
> Could you please also:
> 
> * [ ]  Illustrate how to do with with `twine`
> * [ ]  Add a page in the wiki for this
> 
> Thanks! :)

Created a PR in strongbox-docs for wiki page. :)

**Pull request merged**